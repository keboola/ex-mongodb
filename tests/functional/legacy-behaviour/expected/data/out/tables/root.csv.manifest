{"incremental":false,"columns":["id","level1"],"column_metadata":{"id":[{"key":"KBC.datatype.nullable","value":true}],"level1":[{"key":"KBC.datatype.nullable","value":true}]}}