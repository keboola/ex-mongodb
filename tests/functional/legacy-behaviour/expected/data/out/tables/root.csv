"5716054bee6e764c94fa7aaa","7d0cae8a5690da82b226698d23e94a21"
"5716054bee6e764c94fa7aab","dc6b61323e318be5ec4a277583bdfe0b"
"5716054bee6e764c94fa7aac","81f804cf4fb0e85c4a6563ce5184c92d"
"5716054bee6e764c94fa7aad","0e758a1da8bc9357e64286b1d0fbe2a0"
